Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -2574,7 +2574,7 @@ void @prototype_class@::initialize(JS::Realm& realm)
     }
 
     generator.append(R"~~~(
-    define_direct_property(*vm.well_known_symbol_to_string_tag(), js_string(vm, "@name@"), JS::Attribute::Configurable);
+    define_direct_property(*vm.well_known_symbol_to_string_tag(), PrimitiveString::create(vm, "@name@"), JS::Attribute::Configurable);
```

Comment:
This needs a `JS` think ADL saved us here until now? (Although I would expect that to still work but anyway that's C++)

Reply:
Whoops! TIL about ADL